google/cloud/storage/bucket.py

```diff
@@ -374,6 +374,36 @@ def from_api_repr(cls, resource):
         return instance
 
 
+class LifecycleRuleAbortIncompleteMultipartUpload(dict):
+    """Map a rule aborting incomplete multipart uploads of matching items.
+
+    :type kw: dict
+    :params kw: arguments passed to :class:`LifecycleRuleConditions`.
```

Comment:
Could we add in the docstrings that the Age Lifecycle condition is the only allowed condition for this action?